src/Controller/Controller_Zapier.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace GFPDF\Controller;
+
+use GFPDF\Helper\Helper_Url_Signer;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2022, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ */
+
+/* Exit if accessed directly */
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Controller_Zapier
+ *
+ * @package GFPDF\Controller
+ */
+class Controller_Zapier {
+
+	private $log;
```

Comment:
@jestonihpi unless we really need to prevent classes that extend this one from accessing the $log property, we use `protected` and not `private`